ukwa/npld-player
# Webview - Visit initial web address, go back, forward, home

https://github.com/ukwa/npld-player/issues/7, https://github.com/ukwa/npld-player/issues/5 Enables users to go back, forward, and home from initially visited web address.

### Manual testing
1. Prerequisite:
  a. Create `.env` file from `sample.env` (`cp sample.env .env`)
  b. Run `yarn` to install new dependencies
  c. Run `yarn start` to open app
1. Click links within website. Verify address bar updates as expected
1. Click arrows and home icon to go back/forward/home. Verify app updates as expected
1. Minimize window size until URL is truncated. Verify you're able to focus on the address bar and scroll to the end of the address bar to view the full URL

### Screenshots
<img width="1114" alt="Screen Shot 2022-05-06 at 1 32 37 PM" src="https://user-images.githubusercontent.com/4672952/167212640-20c90ecb-b9ca-4860-903e-a0033c1dadae.png">

### TODO
- Handle external links/links opened in new window
- Handle prefix/protocol
